### PR TITLE
Desktop version of my portfolio

### PR DESCRIPTION
**Pull request by NoelLincoln**
**Hello 👋**
I have finished the desktop design as required  : 
- using flexbox layout 
- replicating the figma design as much as possible
- using percentages instead of absolute value like pixels

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -796,4 +781,133 @@ input[type='submit']:active {
...
+    width: 100%;
+    background: #dfe1e6;
+    border: none;
+  }
+
+  .contact-form {
+    padding-left: 20%;
+    padding-right: 20%;
+    background-image: url(assets/images/contact-form-bg.svg);
+  }
```
Comment (zieeco) [inline style.css:922]: - Hey, please, let's control the extra white space here by removing it. see the image below:

image

File: style.css [hunk 2]
```diff
@@ -796,4 +781,133 @@ input[type='submit']:active {
...
+
+  .desktop-menu li a {
+    color: #344563;
+    text-align: center;
+    font-family: Poppins, sans-serif;
+    font-size: 15px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 20px;
+    text-decoration: none !important;
```
Comment (zieeco) [inline style.css]: - Hey, let's try to add the rule without using the !important rule as a guide in the microverse list of the [HTML & CSS requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). Plese, kindly remove it.

File: style.css [hunk 3]
```diff
@@ -796,4 +781,133 @@ input[type='submit']:active {
...
+
+  .top {
+    width: 100%;
+    max-width: 50%;
+    margin-left: 0;
+    margin-right: 0;
+  }
+
+  .top h2,
+  p,
```
Comment (zieeco) [inline style.css:881]: -- Hey, let's improve the font sizes of the elements in the desktop version as yours is still not close. below is an example of the paragraph in the headline section. please, improve on all the font sizes to make your design better:


image

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @NoelLincoln,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions

- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -796,4 +781,145 @@ input[type='submit']:active {
...
+
+  .desktop-menu li a {
+    color: #344563;
+    text-align: center;
+    font-family: Poppins, sans-serif;
+    font-size: 15px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 20px;
+    text-decoration: none !important;
```
Comment (Ibizugbe) [inline style.css]: Just as the previous reviewer has requested, Kindly take away every `!important` rule in your project as this is part of the  [HTML & CSS requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) requirements👍.

image

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @NoelLincoln 👋👋, 

tryagain


Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights✅✅

- [x] Font-sizes has been improved as requested by the previous reviewer👍.
- [x] White space has been removed as requested by the previous reviewer👍.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [OPTIIONAL] Nice job so far👍. there are still some improvements that can be added to your project to make it match the design😉 I think the previous reviewer probably forgot to point them out.
- [OPTIONAL] for example, the texts in the project cards on your desktop are going out of position, both on mobile and desktop.
yours
image

design
image

- [OPTIONAL]Also, Kindly note that the reverse properties in your cards only apply to desktop screens and not mobile screens🥲.
Yours 
image

design
image

Note that on mobile, the image has to always come first, on like in the desktop screen😉.

- [OPTIONAL] In summary, Kindly compare your project to the design and ensure everything looks similar as  adhering strictly to a design is a great professional skill in the job market and it makes you more attractive to employers😉.


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @NoelLincoln     , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights :fire: 
- All requested changes implemented :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**